### PR TITLE
Update License Banners according to Eclipse guidelines

### LINKED ISSUE #94: [PBI] Update License Banners according to Eclipse guidelines
## Description

The license banners at the top of all files need to be updated according to the Eclipse guidelines.

LICENSE identifier should follow Eclipse Guidelines:

Suggested example from Eclipse:

```
// Copyright (c) Microsoft Corporation. 
// Licensed under the MIT license.
// SPDX-License-Identifier: MIT
```

The first occurrence of the project name in every file should be "Eclipse Chariott", not "Chariott" 

## Acceptance criteria
- [ ] All License banners match the above format
- [ ] The first occurrence of the project name in every file appears as "Eclipse Chariott"

### PR DESCRIPTION
**Pull request by ladatz**
Closes #94 

## Description
Update License Banners according to Eclipse guidelines. The following is a sample of what they should look like:

`// Copyright (c) Microsoft Corporation.  

// Licensed under the MIT license.

// SPDX-License-Identifier: MIT`

**PR COMMIT MESSAGE**

<type>docs: Update License Banners according to Eclipse guidelines </description>

### CODE REVIEW

File: tests/container-e2e-tests-wsl2.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
```
Comment (devkelley) [inline tests/container-e2e-tests-wsl2.sh:1]: You had updated '#!/bin/bash' to '#!/usr/bin/env bash' elsewhere. Does this need to change too?

File: build-all-containers.sh [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
```
Comment (devkelley) [inline build-all-containers.sh:1]: Same comment about '#!/bin/bash' as one below

File: tests/container-e2e-tests-wsl2.sh (hunk 1 above)
Comment (ladatz) [reply tests/container-e2e-tests-wsl2.sh:1]: The other file was an unintentional change. I will leave this one as-is and revert the other.

File: build-all-containers.sh (hunk 2 above)
Comment (ladatz) [reply build-all-containers.sh:1]: The other file was an unintentional change. I will leave this one as-is and revert the other.

**Pull request merged**